### PR TITLE
Serializers test

### PR DESCRIPTION
**Pull request by sonalkr132**
So far, 0% coverage gain. Yay!

### CODE REVIEW

File: app/serializers/event_serializer.rb [hunk 1]
```diff
@@ -5,7 +5,7 @@ class EventSerializer < ActiveModel::Serializer
 
   def date
     t = object.start_time
-    t.blank? ? '' : %{ #{l t, format: :short}#{t.formatted_offset(false)} }
+    t.blank? ? '' : %{ #{I18n.l t, format: :short}#{t.formatted_offset(false)} }
```
Comment (nishanthvijayan) [inline app/serializers/event_serializer.rb]: I think you can remove t.formatted_offset(false) and add it to :short  by adding ":z" .I'm not sure, so please do verify they give the same output.

Comment (kalabiyau) [inline app/serializers/event_serializer.rb]: @nishanthvijayan I would recommend to look at the very last comment in #904 about that case

Comment (hennevogel) [inline app/serializers/event_serializer.rb]: This is out of scope for this PR....

**Comment by hennevogel** [comment]
Awesome work. Thanks
